packages/cli/src/commands/deploy/index.ts

```diff
@@ -194,6 +235,21 @@ export default async (client: Client) => {
       );
       return 1;
     }
+
+    const prebuiltBuild = fs.readJsonSync(
```

Comment:
Not a huge deal, but for consistency let's stick with async operations.

```suggestion
    const prebuiltBuild = await fs.readJSON(
```